### ISSUE TITLE
Отсутствие некоторых форм глаголов

### ISSUE THREAD

**Issue by zanud** [sokirko74/aot#8]
В языке (и словарях) имеются четвёрки (пары пар) глаголов вроде:
```
обнадёживать, обнадёживаться
обнадёжить, обнадёжиться
```

Первая пара означает выполнение действия, а вторая - его совершение (окончание).
Но иногда попадаются  глаголы, у которых в одной из пар второе слово отсутствует. Например:
```
обезнадёживать, обезнадёживаться
обезнадёжить, -
```
или
```
перекорёживать, перекорёживаться
перекорёжить, -
```
Причём этих отсутствущих глаголов нет даже в словарях, изданных Институтом русского языка!
Но я логики в этом не вижу: действие выполняется и вполне может быть завершено. И слова "обезнадёжиться" и "перекорёжиться" для меня звучат вполне естественно.

Нормально ли будет добавить подобные глаголы в словарь?

**Comment by bzaar** [comment]
А не проще ли включить формы на -ся в парадигму переходных глаголов без -ся? Т.е. например, **перекорёжить** будет включать *перекорёжиться, перекорёжилось* и т.п. как формы.

У Зализняка именно так: 

https://gramdict.ru/conjugation/symbols#personal-forms-sya
https://gramdict.ru/conjugation/symbols#nonpersonal-forms-sya

**Comment by zanud** [comment]
> У Зализняка именно так

А в словаре Лопатина (= [официальный словарь Института русского языка](http://orfo.ruslang.ru/)) даже причастия из глаголов в отдельные статьи вынесены (хе-хе).

Теперь серьёзно.
С точки зрения простоты устройства словаря разницы почти нет: в одном случае мы имеем два набора правил (один - для глаголов, имеющих формы на -ся, и другой - для глаголов, таких форм не имеющих), а во втором случае - тоже два набора (один - для глаголов не на -ся, и другой - для глаголов на -ся). И в обоих случаях при внесении глагола в словарь чешем в затылке: есть у него форма на -ся или нет?
Разница будет только в количестве словарных статей в словаре, но это мелочь. (Хотя те же системы проверки правописания от более компактного словаря могут и выиграть. Не проверял.)

Мой же вопрос более радикальный: в какой мере можно позволить себе "исправлять упущения" официальных словарей? Два раза я себе уже такую вольность позволил, но там мне удалось опереться на старые словари (один аж XIX века), а это редкое везение.

**Comment by bzaar** [comment]
Если словарь отмечает переходность (как ГС Зализняка), то поводов чесать в затылке на один меньше: если переходный, то автоматически появляются формы на -ся.

**Comment by zanud** [comment]
Ага, только вон вверху приведены примеры, когда четвёрного глагола в официальном комплекте нет. И причину этого я не понимаю. А хочу понимать.

Из свежайшего: в нынешних официальных орфографических словарях есть "перехлёстывать", но нет "перехлёстываться". А я почесал в затылке и из некоторых своих соображений, решил, что такое слово есть. А потом нашёл его в толковом словаре Ушакова. А ещё потом нашёл его у Даля (причём с ровно тем же обоснованием, которое перед этим сам придумал :-).

Но, повторяю, в словаре Института русского языка варианта "перехлёстываться" нет. Так вот имею я моральное право вносить это слово в словари или нет?

**Comment by zanud** [comment]
Насчёт "перехлёстываться" я перепутал (не удивительно, когда за день анализируешь порядка 2 тысяч слов). Там  другая "страдательная" история - от "перехлёстывать" в словарях, в том числе и у Зализняка, нет страдательных причастий. А я выяснил, что должны быть.

**Comment by bzaar** [comment]
[У Зализняка](https://gramdict.ru/search/%D0%BF%D0%B5%D1%80%D0%B5%D1%85%D0%BB%D1%91%D1%81%D1%82%D1%8B%D0%B2%D0%B0%D1%82%D1%8C) перехлёстывать непереходный:

> перехлёстывать нсв нп 1a ◑3(-ну́ть)
> перехлёстываться нсв 1a ◑3(-ну́ться)

[В Викисловаре](https://ru.wiktionary.org/wiki/%D0%BF%D0%B5%D1%80%D0%B5%D1%85%D0%BB%D1%91%D1%81%D1%82%D1%8B%D0%B2%D0%B0%D1%82%D1%8C) переходность явно не отмечена:

1. сильным всплеском, волной переливаться через край чего-либо, через кого-, что-либо (о воде) 
2. перен., разг., неодобр. проявлять неумеренность в чём-либо, утрачивать чувство меры; проявляться с излишней силой (о каком-либо чувстве, переживании и т. п.)
3. разг. туго перетягивать, стягивать
4. разг. пересекать что-либо

Переходным он может быть в значениях 3 и 4, оба разговорные, поэтому у Зализняка их нет (словарь нормативный).

**Comment by zanud** [comment]
Вот, все нынешние словари оперируют, в основном, пунктами 1 и 2. А они, на самом деле, - не основное значение слова, а побочное (причём 2 явно произошло от 1).
А изначальное значение, от которого они ответвились, вот - [у Даля](https://dic.academic.ru/dic.nsf/enc2p/305922):
> ПЕРЕХЛЕСТЫВАТЬ, перехлеснуть веревку, перекинуть и захлеснуть, затянуть

Верёвка, когда её перехлёстывают, является перехлёстываемой. И предмет, через который её перебрасывают, является перехлёстываемым.

**Comment by bzaar** [comment]
Ну, вот мы с Вами и разобрались :) Т.е. это дело каждого составителя словаря – выбирать, какую стилистику включать *(разг., устар.)*  Ясно, что границы между ними далеко не всегда четкие. Так что лексикография – довольно субъективное занятие.

**Comment by bzaar** [comment]
Кстати, в ruscorpora [не нашел](https://processing.ruscorpora.ru/search.xml?env=alpha&api=1.0&mycorp=&mysent=&mysize=&mysentsize=&dpp=&spp=&spd=&mydocsize=&mode=main&lang=ru&sort=i_grtagging&nodia=1&text=lexgramm&parent1=0&level1=0&lex1=%D0%BF%D0%B5%D1%80%D0%B5%D1%85%D0%BB%D0%B5%D1%81%D1%82%D1%8B%D0%B2%D0%B0%D1%82%D1%8C&gramm1=partcp&sem1=&flags1=&sem-mod1=sem&sem-mod1=sem2&parent2=0&level2=0&min2=1&max2=1&lex2=&gramm2=&sem2=&flags2=&sem-mod2=sem&sem-mod2=sem2) ничего «перехлёстываемого».

**Comment by zanud** [comment]
> Т.е. это дело каждого составителя словаря – выбирать, какую стилистику включать

Совершенно верно, и я об этом ранее тоже писал. И один из моих вопросов: какова политика наполнения этого словаря? (Я же новичок здесь.)

А второй вопрос: что делать со словами, которые отсутствуют в словарях, но, по моему мнению, вполне себе существуют (например, "обезнадёжиться" и "перекорёжиться" из первого сообщения)?

**Comment by zanud** [comment]
> Кстати, в ruscorpora не нашел ничего «перехлёстываемого».

За последние полтора месяца я перелопатил около 150 тысяч слов (включая словоформы). Все они присутствуют в словаре и 95% из них вполе себе литературны. Но я уверен, что существенная их часть никогда ни в каких текстах не фигурировала. Однако это же не основание для невключения их в словари.

Ой, да, а ещё в моих списках есть изрядное количество отглагольных существительных, которых нет в словарях. Вот так - глагол в словаре есть, а существительного, означающего это действие, почему-то нет. А должно бы быть.

**Comment by bzaar** [comment]
Уважаемый @zanud, снова здраствуйте!

> какова политика наполнения этого словаря?

За политику АОТ не поручусь (это вопрос к @sokirko74), а для себя (при составлении odict.ru) я использую критерий: если слово встречается хотя бы в трех разных источниках, то оно «существует», т.е. достойно попасть в словарь.

Причем в источнике не должна идти речь только о самом слове («Как правильно, екатеринбуржка или екатеринбурженка?»), слово должно быть употреблено ради своего значения, а не потому что на него упал пытливый взор лингвиста. Это автоматом исключает словари (некоторые слова только в словарях и живут).  Но словари могут быть полезны, если в них есть примеры и ссылки на первоисточники (как в Викисловаре).

**Comment by bzaar** [comment]
Конечно, у Викисловаря есть недостатки: писали его Шарик с Матроскиным и выкачивается он часами. Поэтому некоторые не стесняются своих лексикографических порывов и [составляют словари](https://github.com/Pogrebnoj-Alexandroff/demonyms/blob/patch-1/-jets.tsv) прямо на Гитхабе.
